lua/ulx/modules/sh/cleanropes.lua

```diff
@@ -0,0 +1,18 @@
+local cleanRopes = ulx.command(
+	TASUtils.Category,
+	"ulx cleanropes",
+	function(calling_ply)
+		local ropes = ents.FindByClass("keyframe_rope")
+		for _, v in ipairs(ropes) do
+			if v.Ent1 and v.Ent1:IsWorld() and v.Ent2 and v.Ent2:IsWorld() then
+				v:Remove()
+			end
+		end
+
+		ulx.fancyLogAdmin(calling_ply, "#A cleaned up #i world ropes", #ropes)
```

Comment:
This length with include the ropes that weren't removed when the condition above is false